### PR TITLE
Creation of API readiness checklist document (Issue 39)

### PR DESCRIPTION
**Pull request by jpengar**
#### What type of PR is this?

* documentation
* subproject management

#### What this PR does / why we need it:

This PR creates a document with the API readiness checklist status according to CAMARA Commonalities.

#### Which issue(s) this PR fixes:

Fixes one of the topics to be resolved in Issue https://github.com/camaraproject/HomeDevicesQoD/issues/39

#### Special notes for reviewers:

N/A

#### Changelog input

```
 Creation of API readiness checklist document

```

### CODE REVIEW

File: documentation/API_documentation/home_devices_qod_readiness_checklist.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# HomeDevicesQoD API Readiness minimum criteria checklist
+
```
Comment (rartych) [inline documentation/API_documentation/home_devices_qod_readiness_checklist.md:2]: Would the link to the template https://github.com/camaraproject/WorkingGroups/blob/main/Commonalities/documentation/API-Readiness-Checklist.md be helpful for the reader?

File: documentation/API_documentation/home_devices_qod_readiness_checklist.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+<br>
+
+| No | Deliverables/Criteria            | Mandatory | Status (Contributed/Approved/Pending/Validated/Partly-Validated)|
+|----|----------------------------------|-----------|----------------------------                         
+|  1 |API Spec                          |   Y       | Contributed                |
+|  2 |API Implementation                |   N       |                            |
+|  3 |API Documentation                 |   Y       | Contributed                |
+|  4 |User Stories                      |   Y       | Contributed                |
```
Comment (rartych) [inline documentation/API_documentation/home_devices_qod_readiness_checklist.md:10]: Could links to User stories/test etc. be added in this column?

File: documentation/API_documentation/home_devices_qod_readiness_checklist.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+|  5 |API test cases                    |   Y       | Pending (TBC)              |
+|  6 |Validated by atleast 2 operators  |   Y       | Pending (TBC)              |
```
Comment (rartych) [inline documentation/API_documentation/home_devices_qod_readiness_checklist.md:12]: Typo in the template: **at least**

File: documentation/API_documentation/home_devices_qod_readiness_checklist.md (hunk 1 above)
Comment (jpengar) [reply documentation/API_documentation/home_devices_qod_readiness_checklist.md:2]: Resolved in https://github.com/camaraproject/HomeDevicesQoD/pull/40/commits/27cb83819023aa630814b1a54a17955588d4c694

File: documentation/API_documentation/home_devices_qod_readiness_checklist.md (hunk 2 above)
Comment (jpengar) [reply documentation/API_documentation/home_devices_qod_readiness_checklist.md]: Resolved in https://github.com/camaraproject/HomeDevicesQoD/pull/40/commits/27cb83819023aa630814b1a54a17955588d4c694

File: documentation/API_documentation/home_devices_qod_readiness_checklist.md [hunk 4]
```diff
@@ -0,0 +1,13 @@
+# HomeDevicesQoD API Readiness minimum criteria checklist
+
+<br>
+
+| No | Deliverables/Criteria            | Mandatory | Status (Contributed/Approved/Pending/Validated/Partly-Validated)|
```
[the remark this answers is not in the document]
Comment (jpengar) [reply documentation/API_documentation/home_devices_qod_readiness_checklist.md]: Resolved in https://github.com/camaraproject/HomeDevicesQoD/pull/40/commits/27cb83819023aa630814b1a54a17955588d4c694

File: documentation/API_documentation/home_devices_qod_readiness_checklist.md (hunk 3 above)
Comment (jpengar) [reply documentation/API_documentation/home_devices_qod_readiness_checklist.md]: Resolved in https://github.com/camaraproject/HomeDevicesQoD/pull/40/commits/27cb83819023aa630814b1a54a17955588d4c694

**Review by rartych — APPROVED** [review]
LGTM.
I was not aware it is based on QoD document, but this is more readable now.
Thanks!

**Review by monamok — APPROVED** [review]

**Pull request merged**